### ISSUE TITLE
user_script:70: too many results to unpack

### ISSUE THREAD

**Issue by yixiangfeng** [OptimalBits/bull#2709]
## Description
my job queue processor some time can complete and no throw the unpack error. after few time the job will return the unpack error.
<img width="1209" alt="螢幕截圖 2024-02-19 11 10 22" src="https://github.com/OptimalBits/bull/assets/55069287/6f3694ca-511a-4709-bb51-0f79df5d5c27">
 

## Bull version
v4.10.4
## Additional information
my redis have 19 queues
some queues will have repeat job

how to solve the problem

**Comment by matt-moven** [comment]
We also ran into this bug and traced it down to the `collectMetrics` function in "moveToFinished-9.lua". We set `metrics.maxDataPoints` to 1. In `moveToFinished`, `collectMetrics` tries to add the current metric and also fill in zero values for all the missed seconds, and then trims that list down to `maxDataPoints`. If a queue sees no jobs for enough time,  and then processes a job and tries to update its metrics, the `unpack()` with one metric and many zeros fails with error "too many results to unpack".

https://github.com/OptimalBits/bull/blob/886d7643819dcf52902d2e92394267dbd495c71b/lib/commands/moveToFinished-9.lua#L70
